packages/opentelemetry-sampler-aws-xray/src/remote-sampler.types.ts

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
I assume the name of this file is based on the aws-sdk instrumentation which has the `aws-sdk.types.ts`. The reason it is called that way is because this file contains types copied as is from the `aws-sdk` npm package. 

If those types are copied from the `aws-sdk` package as well, I recommend calling it `aws-sdk.types.ts` to be consistent. If not, the convention name as used in other packages is `types.ts` or `internal-types.ts`, depending on wether you want them to be public or not